.builds/alpine.yml

```diff
@@ -19,7 +19,7 @@ tasks:
   # version, instead of master, to avoid any breaking changes in wlroots.
   - wlroots: |
       cd wlroots
-      git checkout 0.13.0
+      git checkout 0.14.0
```

Comment:
Tip: you can append `#0.14.0` in `sources` to achieve the same thing, if you prefer that notation.

Reply:
Neat, I didn't know that! Thanks for the tip :)